Library/Formula/eg.rb

```diff
@@ -0,0 +1,40 @@
+class Eg < Formula
+  homepage "https://github.com/srsudar/eg"
+  url "https://github.com/srsudar/eg/archive/v0.0.3.tar.gz"
+  sha256 "c257e334f2a2e494f9b9791b92c1d3f7b00da3e2e7ec47f02d720342b466f968"
+
+  def install
+    if Dir.exist? "#{var}/examples"
```

Comment:
No, we're saying `eg`'s examples should not be installed into `var` as it won't be remove on uninstall of `eg` if put there (or `etc`)